tests/FrameCameraTests.cpp

```diff
@@ -13,6 +13,61 @@
 using namespace std;
 using json = nlohmann::json;
 
+
+class FramerParameterizedTest : public ::testing::TestWithParam<csm::ImageCoord> {
```

Comment:
I think we should parameterize based on the keyword in the ISD being modified instead of the image point.

Reply:
That would help clean up many of Kaj's tests, but I can think of plenty of situations where we want to parameterize on the values a keyword can take (i.e. Checking some distortion function with different values for a focal length, or passing in a range of double vectors for the keywords odt_x and odt_y).  My goal here was to put in something that is as easy to follow for others as possible. Parameterizing on an image pt was about as simple as I could make it.

Reply:
:+1: that sounds reasonable

Reply:
Agreed. I'm imagining we select 3-5 test points and run image to ground/ ground to image on each of those tests points and parameterize by the ISD keyword.

So, we'd have tests called "CenterPixel", "OffCenter", "OffBody", etc.

Then we would parameterize using pairs of ISD keywords and values.

Reply:
I think this decision is going to be up to whomever does this next FY

Reply:
That sounds good.